### ISSUE TITLE
Mystique causes fatal error on pages without template context

### ISSUE THREAD

**Issue by flipzoom** [trk/Mystique#5]
On pages without template context, such as the user templates, Mystique causes a fatal error. Here is a quick workaround. Maybe you have a better solution for the problem? 

## Stack trace
```
2020-05-18 14:50:26 admin http://localhost/testing/pw-fzmcms-xdebug/fzmcms/access/users/

Fatal Error: Uncaught Error: Call to a member function getField() on null in C:\xampp\htdocs\testing\pw-fzmcms-xdebug\site\modules\Mystique\FieldtypeMystique.module.php:71 

Stack trace: 

#0 C:\xampp\htdocs\testing\pw-fzmcms-xdebug\wire\modules\Inputfield\InputfieldSelector\InputfieldSelector.module(1052): ProcessWire\FieldtypeMystique->getBlankValue(Object(ProcessWire\NullPage), Object(ProcessWire\Field)) 

#1 C:\xampp\htdocs\testing\pw-fzmcms-xdebug\wire\modules\Inputfield\InputfieldSelector\InputfieldSelector.module(941): ProcessWire\InputfieldSelector->renderSelectFieldOption(Object(ProcessWire\Field), Array, 'name') 

#2 C:\xampp\htdocs\testing\pw-fzmcms-xdebug\wire\modules\Inputfield\InputfieldSelector\InputfieldSelector.module(1958): ProcessWire\InputfieldSelector->renderSelectField(Array, 'name') 

#3 C:\xampp\htdocs\testing\pw-fzmcms-xdebug\wire\core\Wire.php(380): ProcessWire\InputfieldSelector->___render() 

#4 C:\xampp\htdocs\testing\pw-fzmcms-xdebug\wire\core\WireHooks.php(823): ProcessWire\Wire->_callMethod('___render' (line 71 of C:\xampp\htdocs\testing\pw-fzmcms-xdebug\site\modules\Mystique\FieldtypeMystique.module.php)
```

Change [line 71](https://github.com/trk/Mystique/blob/master/FieldtypeMystique.module.php#L71) in the file FieldtypeMytique.module.php from: 
```php
return new MystiqueValue($page, $page->template->fieldgroup->getField($field,true));
```
to the following: 
```php
return ($page instanceof NullPage) ? new MystiqueValue($page, $field) : new MystiqueValue($page, $page->template->fieldgroup->getField($field, true));
```

**Comment by trk** [comment]
Do you still have this error on latest version ?

**Comment by flipzoom** [comment]
Fixed with version 0.0.15. THX.
